openreview/venue/process/submission_process.py

```diff
@@ -37,11 +37,12 @@ def process(client, edit, invitation):
 To view your submission, click here: https://openreview.net/forum?id={note.forum}'''
     
     #send tauthor email
-    client.post_message(
-        subject=author_subject,
-        message=author_message,
-        recipients=[edit.tauthor] if edit.tauthor != 'OpenReview.net' else []
-    )
+    if edit.tauthor != 'OpenReview.net':
```

Comment:
```suggestion
    if edit.tauthor.lower() != 'openreview.net':
```

Reply:
super user for the test environment is `openreview.net` in lowercase. Using this change you can add a test and verify that no emails is being sent then the submission is being posted by the super user. Could you add this test?